using_images/xpaas_images/a_mq.adoc

```diff
@@ -5,4 +5,92 @@
+
+== Differences Between the JBoss A-MQ xPaaS Image and the Regular Release of JBoss A-MQ
+
+There are several major functionality differences in the OpenShift JBoss A-MQ xPaaS image:
+
+* The Karaf shell is not available.
+* The Fuse Management Console (Hawtio) is not available.
+* Configuration of the broker can be performed:
+** using parameters specified in the A-MQ application template, as described in xref:configuring-params[Application Template Parameters].
+** using the STI (Source-to-image) tool, as described in xref:configuring-sti[Configuration Using STI].
+
+
+== Installing the JBoss A-MQ xPaaS Images
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must first install the image streams and source-to-image (S2I) application templates.
+
+. In a terminal, change into the root directory of your OpenShift master. It should contain the 'examples' subdirectory.
+. Create the image stream from which the A-MQ image will be obtained:
++
+----
+# oc create -f examples/application-templates/jboss-image-streams.json -n <project>
+----
+. Create the S2I templates for your A-MQ application:
++
+----
+# oc create -f examples/application-templates/amq/amq6.json -n <project>
+# oc create -f examples/application-templates/amq/amq6-persistence.json -n <project>
+----
+
+You will now have the JBoss A-MQ xPaaS images stream and S2I application templates available for use in your OpenShift project. 
+
+To create an application based on the template, follow instructions in the *Creating a Configuration From a Template* section of the link:../dev_guide/templates.html[Creating New Applications] topic in the OpenShift documentation. For a reference on template parameters specified when creating the application, see the xref:configuring-params[Application Template Parameters] section below.
+
+== Configuring the JBoss A-MQ Image
+
+[[configuring-params]]
+=== Application Template Parameters
+
+Basic configuration of the JBoss A-MQ xPaaS image is performed by specifying values of application template parameters. The following parameters can be configured:
+
+AMQ_RELEASE::
+  The JBoss A-MQ release version. This determines which JBoss A-MQ image will be used as a basis for the application. At the moment, only version _6.2_ is available.
+APPLICATION_NAME::
+  The name of the application used internally in OpenShift. It is used in names of services, pods and other objects within the application.
+MQ_USERNAME::
+  The user name used for authentication to the broker. In a standard non-containerized JBoss A-MQ, you would specify the user name in the 'AMQ_HOME/opt/user.properties' file. If no value is specified, a random user name is generated.
```

Comment:
Markup for file names/directories:

```
*_AMQ_HOME/opt/user.properties_*
```

See: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference